DigitalLearningSolutions.Web/ViewModels/Login/LoginViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Login
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class LoginViewModel
+    {
+        public LoginViewModel()
+        {
+            Username = string.Empty;
+            Password = string.Empty;
+            RememberMe = false;
+        }
+
+        [Display(Order = -1)]
```

Comment:
Does this actually fix the ordering issue/how does it work when combined with the errors added in controller?

Reply:
Short answer: it doesn't do anything to the error summary, and I forgot to remove it.